### PR TITLE
feat(ecma/parser): add the `allow_return_outside_function` option

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Add `allow_return_outside_function` to parse HTML event handler attributes

**BREAKING CHANGE:**

Yes

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_html_minifier/src/lib.rs [hunk 1]
```diff
@@ -1069,6 +1073,13 @@ impl Minifier {
             }
         };
 
+        println!("{:?}", errors);
```
Comment (kdy1) [inline crates/swc_html_minifier/src/lib.rs:1076]: Seeems like a mistake

**Comment by alexander-akait** [comment]
@kdy1 Fixed and rebase

File: crates/swc_ecma_parser/src/lib.rs [hunk 2]
```diff
@@ -256,6 +256,16 @@ impl Syntax {
         }
     }
 
+    pub fn allow_return_outside_function(self) -> bool {
```
Comment (kdy1) [inline crates/swc_ecma_parser/src/lib.rs:259]: Please reduce visibility of this

**Comment by alexander-akait** [comment]
@kdy1 Fixed

**Review by kdy1 — APPROVED** [review]
Thanks!

---

swc-bump:
 - swc_ecma_parser --breaking

**Pull request merged**